### DISCUSSION TITLE
Can you start a Discord to help out people like me?

### DISCUSSION THREAD

**Discussion by BurnDivision** [scionaltera/agonyforge#168] [Ideas]
I really want to try and get this up and running and test it out, but I think I need more direct help than this.

**Comment by scionaltera** [comment]
Yes, I can work on setting up a new Discord sometime this week. In the meantime though, I wrote some initial documentation that should be helpful. I definitely uncovered a few steps that I had forgotten about when I worked through it. Thanks for sticking with it.

https://github.com/scionaltera/agonyforge/wiki/How-to-Develop

There's a ton of other documentation I'd like to write too, like some in depth discussion of the architecture and how I would set up a "production" version rather than a local development version. But I think this was a good place to start.

**Reply by BurnDivision** [reply]
Well I really want to work on an RPG of my own but I have very little graphical abilities. Writing out some text would be easier in my mind. 

I have a lot of experience with MajorMUD (mostly playing, some editing/hacking) so I thought I'd try my hand at using something hopefully similar. And I like that it's in active development. So hopefully this may work out for me if I can get it running properly.

**Comment by scionaltera** [comment]
I have created a Discord. There is a badge for it at the top of the readme that should work as an invite link.
